shared/common-adapters/button.desktop.js

```diff
@@ -13,6 +13,9 @@ class Button extends Component {
     let backgroundStyle = {}
     let labelStyle = {}
     let progressColor = globalColors.white
+    let rippleStyle: {rippleColor?: string} = {}
```

Comment:
@chrisnojima Whoops, holdover from a previous plan I had. I'm condensing that now.